Sources/Purchasing/Purchases/PurchasesOrchestrator.swift

```diff
@@ -1129,6 +1129,13 @@ private extension PurchasesOrchestrator {
                         return
                     }
 
+                    // We dont have an AppTransactionJWS, return the cached CustomerInfo
+                    // even if it does not have originalPurchaseDate and originalApplicationVersion
+                    if let cachedCustomerInfo, appTransactionJWS == nil {
+                        completion?(.success(cachedCustomerInfo))
+                        return
+                    }
+
```

Comment:
this is kind of like a rough equivalent to the missing receipt error from SK1, maybe we should return an error instead?

Reply:
from slack convo: this should only happen in sandbox and when there are no purchases to restore in any case, so returning cached customer info would work from a dev perspective. We should add a log or something to inform the dev that the receipt wasn't actually posted